.changeset/many-jobs-teach.md

```diff
@@ -0,0 +1,9 @@
+---
+"@digdir/designsystemet-css": patch
+---
+
+**Details:** content using `overflow: hidden` to prevent margin clipping
+**Tabs:** adjusted to allow non-tab-button-elements inside `tablist`
+**Table:**
+- sorting icon never shrinks, even when using `Button` component (which is `display: flex`) as parent
+- with `data-border` adjusts `border-radius` according to `border-width`
```

Comment:
I think these should be separate changesets, but we can test if they are formatted correctly...